### PR TITLE
Lt 912 add an external note to a case

### PR DESCRIPTION
**Pull request by 5pence**

### CODE REVIEW

File: ui_automation_tests/conftest.py [hunk 1]
```diff
@@ -17,18 +17,18 @@ def pytest_exception_interact(node, report):
...
 def pytest_addoption(parser):
     env = str(os.environ.get('ENVIRONMENT'))
     if env == 'None':
         env = "dev"
     print("touched: " + env)
     parser.addoption("--driver", action="store", default="chrome", help="Type in browser type")
-    parser.addoption("--exporter_url", action="store",
-                      default="https://exporter.lite.service." + env + ".uktrade.io/", help="url")
-    parser.addoption("--internal_url", action="store",
-    # parser.addoption("--exporter_url", action="store", default="http://localhost:8300", help="url")
```
Comment (jfaracik) [inline ui_automation_tests/conftest.py]: Should this file be changed?

File: ui_automation_tests/conftest.py [hunk 2]
```diff
@@ -17,18 +17,18 @@ def pytest_exception_interact(node, report):
         # utils.save_screenshot(node.funcargs.get("driver"), name)
 
 
-# Create driver and url command line addoption
+# Create driver and url command line adoption
 def pytest_addoption(parser):
     env = str(os.environ.get('ENVIRONMENT'))
     if env == 'None':
         env = "dev"
     print("touched: " + env)
```
Comment (cstobart) [inline ui_automation_tests/conftest.py]: Remove or add to logging

File: ui_automation_tests/conftest.py [hunk 3]
```diff
@@ -17,18 +17,18 @@ def pytest_exception_interact(node, report):
...
     parser.addoption("--driver", action="store", default="chrome", help="Type in browser type")
-    parser.addoption("--exporter_url", action="store",
-                      default="https://exporter.lite.service." + env + ".uktrade.io/", help="url")
-    parser.addoption("--internal_url", action="store",
-    # parser.addoption("--exporter_url", action="store", default="http://localhost:8300", help="url")
-    # parser.addoption("--internal_url", action="store", default="http://localhost:8080", help="url")
+    # parser.addoption("--exporter_url", action="store",
+    #                   default="https://exporter.lite.service." + env + ".uktrade.io/", help="url")
```
Comment (cstobart) [inline ui_automation_tests/conftest.py]: reverse commenting so tests run against dev

File: ui_automation_tests/conftest.py (hunk 1 above)
Comment (5pence) [reply ui_automation_tests/conftest.py]: yes have changed it - good catch Jan

File: ui_automation_tests/conftest.py (hunk 2 above)
Comment (5pence) [reply ui_automation_tests/conftest.py]: done

File: ui_automation_tests/conftest.py (hunk 3 above)
Comment (5pence) [reply ui_automation_tests/conftest.py]: done good catch

**Comment by efahy** [comment]
It looks like the guidance text around case note length is not appearing when entering the text of a new note - can you investigate please?

File: templates/cases/case/tabs/activity.html [hunk 4]
```diff
@@ -84,19 +81,23 @@ <h3 class="govuk-heading-m">Goods Location</h3>
...
+					</div>
 					<a id="case-note-cancel-button" href="{% url 'cases:case' data.case.id %}" class="govuk-button govuk-button--secondary" type="button" draggable="false">
 						Cancel
 					</a>
-					<p id="case_note-warning" class="govuk-hint govuk-character-count__message" aria-live="polite">
-						You can enter up to 2200 characters
-					</p>
+<!-- {#					<p id="case_note-warning" class="govuk-hint govuk-character-count__message" aria-live="polite">#}
+{#						You can enter up to 2200 characters#}
+{#					</p>#} -->
```
Comment (efahy) [inline templates/cases/case/tabs/activity.html]: Looks like this is why the note length text isn't appearing - its commented out?

Comment (5pence) [reply templates/cases/case/tabs/activity.html]: This is now sorted out

Comment (5pence) [reply templates/cases/case/tabs/activity.html]: ALSO character count is now working, and e2e test written

**Pull request merged**